csharp/src/AddressBook/AddressBook.csproj

```diff
@@ -41,6 +41,20 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'ReleaseSigned|AnyCPU' ">

```

Comment:
My experience is that sooner or later that's not going to happen, and at least with the different configurations we'd be making sure that unsigned code can still build against it. Basically if we wouldn't *want* to sign something, I'd rather not sign it just for the sake of the 1:1 correspondence... we basically don't have a 1:1 correspondence in requirements :)

I don't feel desperately strongly either, mind you.